package.json

```diff
@@ -95,6 +95,7 @@
   "devDependencies": {
     "@changesets/changelog-github": "0.4.1",
     "@changesets/cli": "^2.26.2",
+    "eslint-plugin-index": "^1.1.5",
```

Comment:
What is the reason for introducing this dependency?

Reply:
Not entirely sure how it's sneaked in, removed